src/Domain/FaceNormal.hpp

```diff
@@ -177,12 +177,10 @@ struct InterfaceCompute<Tags::BoundaryDirectionsExterior<VolumeDim>,
   }
   using return_type = std::unordered_map<::Direction<VolumeDim>,
                                          tnsr::i<DataVector, VolumeDim, Frame>>;
-  static void function(
-      const gsl::not_null<return_type*> normals,
-      const db::const_item_type<Tags::Interface<dirs, Mesh<VolumeDim - 1>>>&
-          meshes,
-      const db::const_item_type<Tags::ElementMap<VolumeDim, Frame>>&
-          map) noexcept {
+  static void function(const gsl::not_null<return_type*> normals,
+                       const std::unordered_map<::Direction<VolumeDim>,
+                                                ::Mesh<VolumeDim - 1>>& meshes,
```

Comment:
(not a change request) Should we switch the type for `Tags::Interface` to `DirectionMap`, and if so, perhaps this would be a good time to do that? It's a bit out of scope for this PR, but related